app/assets/javascripts/send_welcome_sms.js

```diff
@@ -1,13 +1,14 @@
 $(document).ready(() => {
+  const mainContainer = $(".main-container");
```

Comment:
for jQuery elements, add `$` at the beginning of the variable name, in this contxt: `$mainContainer`